app/services/alerts/adapters/analytics_adapter.rb

```diff
@@ -13,9 +13,10 @@ def produce_report(analysis_object)
 
         variables = if pull_variable_data?(analysis_object)
                       {
-                        template_data: analysis_object.front_end_template_data,
-                        chart_data:    analysis_object.front_end_template_chart_data,
-                        table_data:    analysis_object.front_end_template_table_data
+                        template_data:          analysis_object.front_end_template_data,
+                        chart_data:             analysis_object.front_end_template_chart_data,
+                        table_data:             analysis_object.front_end_template_table_data,
+                        priority_template_data: analysis_object.priority_template_data
```

Comment:
maybe this should be priority_values or something or priority_data as it isn't relevant to the actual templating